odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -7,7 +7,8 @@
     'crm_reveal': 'crm_iap_lead',
     'document': 'attachment_indexation',
     'payment_ogone': 'payment_ingenico',
-    # OCA/...
+    # OCA/hr
+    'hr_skill': 'hr_skills'
```

Comment:
```suggestion
    'hr_skills': 'hr_skill',
```
according to what your PR and commit message say

Reply:
It's the contrary... so the title should be incorrect.